pkg/serverless/appsec/appsec.go

```diff
@@ -103,17 +103,16 @@ func (a *AppSec) Close() error {
 	return nil
 }
 
-// Monitor runs the security event rules and return the events as raw JSON byte
-// array.
-func (a *AppSec) Monitor(addresses map[string]interface{}) (events []byte) {
+// Monitor runs the security event rules and return the events as an slice
+func (a *AppSec) Monitor(addresses map[string]any) []any {
```

Comment:
This should be replaced with a version that allows for ephemeral and permanent addresses to be provided (possibly accepting a `waf.RunAddressData` directly?)